dfget/core/api/uploader_api_types.go

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright The Dragonfly Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package api
+
+import (
+	"time"
+)
+
+// CommonURL wraps the basic fields needed for a request.
+type CommonURL struct {
+	IP      string
+	Port    int
+	Timeout time.Duration
+}
```

Comment:
I think that it may be not a good idea to wrap these fields for a request. Actually, these fields are not parameter of uploader server's APIs. It's better to set them as fields of uploaderAPI struct.